package.json

```diff
@@ -18,33 +18,27 @@
     "type": "git",
     "url": "https://github.com/CesiumGS/cesium-webpack-example.git"
   },
-  "main": "index.js",
   "scripts": {
-    "build": "webpack --config webpack.config.js",
-    "start": "webpack serve --config webpack.config.js",
-    "start:built": "npx http-server -a localhost -p 8080 dist/",
+    "prepare": "husky install",
+    "postinstall": "cd webpack-4 && npm install && cd ../webpack-5 && npm install",
     "eslint": "eslint \"./**/*.js\" \"./**/*.html\" --cache --quiet",
     "prettier": "prettier --write --no-config \"**/*\"",
-    "prettier-check": "prettier --check --no-config \"**/*\""
+    "prettier-check": "prettier --check --no-config \"**/*\"",
+    "start-4": "cd webpack-4 && npm run start",
```

Comment:
Do these `start` and `build` scripts belong at the top level? IMO it's confusing that they change the current directory.

Reply:
Similar to my other comment, I added these mostly for convenience of running/building each version from the root directory. I've updated the docs here a bit to try and highlight what they do better

Reply:
I think adding the explanation helps. For that section though, consider my comment here: https://github.com/CesiumGS/cesium-webpack-example/pull/53/files#r1449276833